codegen_glibmm/templates/proxy.h.templ

```diff
@@ -22,61 +23,44 @@ public:
 
     static Glib::RefPtr<{{ interface.cpp_class_name }}> createForBusFinish (Glib::RefPtr<Gio::AsyncResult> result);
 
+    static Glib::RefPtr<{{ interface.cpp_class_name }}> createForBus_sync (
```

Comment:
No space before `(` (I know, we are a bit inconsistent with it, but that's the direction) :-)

Reply:
Okidok. Done.